design/README.md

```diff
@@ -0,0 +1,54 @@
+# Crossplane Design Documents
+This directory contains all documents informing Crossplane's design. Crossplane
+design documents can be divided into two two kinds:
+
+* One-pagers. A One-pager is a brief document pitching an idea for further
+  experimentation. One-pagers, as the name suggests, are usually around one
+  page long. They provide just enough context to achieve buy-in from Crossplane
+  maintainers.
+* Design documents. A design document is a longer, more detailed design. Design
+  documents should typically be preceeded by a one-pager and/or a good amount of
+  research and experimentation.
+
+In some cases a one-pager is sufficient to move forward with a design. Complex
+or controversial designs will require a more detailed design document. Please
+name your documents appropriately when committing to this directory, i.e.
+`one-pager-my-cool-design.md` or `design-doc-my-cool-design.md`.
+
+All documents committed to this directory _must_ include the following header:
+
+```markdown
+# Document Title
+* Owner: Some Person (person@example.org)
+* Reviewers: Crossplane Maintainers
+* Status: Accepted, revision 1.0
+```
+
+The __document owner__ is the person responsible for stewarding its lifecycle.
+The owner will typically be the original document author, though ownership may
+be handed over to another individual over time. The owner may choose to include
+their email address, but doing so is not mandatory.
+
+The __document reviewers__ are a small, targeted audience. Feedback is _always_
+welcome from any member of the Crossplane community, but feedback from the
+elected reviewers carries extra weight.
+
+The __document status__ reflects the lifecycle of the design. Designs may be
+committed to master at any stage in their lifecycle as long as the status is
```

Comment:
this is saying that a doc can be merged to master while still in the Draft state, right? (e.g. the key stakeholders haven't fully bought in yet, but we want the doc in master for whatever reason)

Reply:
It is. Do you think I should make that clearer?

Reply:
no, i think this is clear enough, i was just making sure of your intent.